mucommander-core/src/main/java/com/mucommander/ui/notifier/NotifierProvider.java

```diff
@@ -116,6 +123,23 @@ public static void displayBackgroundNotification(final NotificationType notifica
         });
     }
 
+    /**
+     * Displays a notification above the main frame that hide after provided time.
+     * If given notification is null or blank it does nothing.
+     *
+     * @param mainFrame the main frame
+     * @param bgColor background color (no alpha supported)
+     * @param fgColor foreground color
+     * @param notification the notification text
+     * @param timeout the time-out in ms after which the notification disappears
+     */
+    public static void displayMainFrameNotification(JFrame mainFrame, String notification,
+                                                    Color bgColor, Color fgColor, long timeout) {
```

Comment:
is it set this way by the formatter? I'd expect the formatter to place `mainFrame` at line 136 and each of the other arguments in a separate line that starts under the 'static' word

Reply:
ad 1. It is being lazy initiated when accessed (i.e. when its setting in Prefs is enabled)
ad 2. initial reformat of that piece was wrong, putting args under "(" - now it places them as you say - I don't know what was wrong with Idea and Eclipse formatter back then :) PR is about to be updated in seconds.